etl/app/server/database.js

```diff
@@ -400,11 +400,17 @@ function getProfileEtl(
     try {
       client.query(`ALTER TABLE ${tableName} SET UNLOGGED`);
       let res = await extract(s3Config);
-      while (res.data !== null) {
+      let chunksProcessed = 0;
+      const maxChunks = maxChunksOverride ?? process.env.MAX_CHUNKS;
+      while (
+        res.data !== null &&
+        (!maxChunks || (maxChunks && chunksProcessed < maxChunks))
```

Comment:
This could be simplified:
```suggestion
        (!maxChunks || chunksProcessed < maxChunks)
```

Reply:
Good call!